boards/silabs/radio_boards/siwx917_rb4338a/siwx917_rb4338a.dts

```diff
@@ -71,14 +71,6 @@
 	};
 };
 
-&dma0 {
-	status = "okay";
-};
-
-&ulpdma {
-	status = "okay";
-};
-
```

Comment:
I believe this should be squashed into the commit that introduced this dts file as per the second bullet point and rule "3" here: https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#pr-requirements

Reply:
Sure